### DISCUSSION TITLE
Can one user be shared for 2 or more móvil device?

### DISCUSSION THREAD

**Discussion by UsuarioFabio** [opengisch/QField#3958] [Q&A]
I find it interesting to exploit the platform as an offline field viewer and browser using the cloud version that allows updating.
The project can be built with one user1 and allow another user2 to view it without permission to edit, so the question is: can user2 be shared by two or more mobile devices or is one user needed for each mobile?

Thank you

**Comment by lucienicolier** [comment]
Hi,

You can use on multiple devices, but you can't be simultaneously connected on 2 devices.

In your case, the most adapted plan will be an organization.

Best
